pkg/kubelet/kubelet_server_journal_linux.go

```diff
@@ -52,6 +52,16 @@ func getLoggingCmd(n *nodeLogQuery, services []string) (cmd string, args []strin
 	if n.TailLines != nil {
 		args = append(args, "--pager-end", fmt.Sprintf("--lines=%d", *n.TailLines))
 	}
+
+	var output string
+	if len(n.Format) > 0 {
+		output = n.Format
+	} else {
+		output = "short-precise"
+	}
+
+	args = append(args, fmt.Sprintf("--output=%s", output))
+
```

Comment:
Reminder to follow the same approach as https://github.com/openshift/kubernetes/pull/2198/commits/e25d066bfbd6c8ee084484ae840b368a0eb769f8#r1944617492